app/models/manageiq/providers/amazon/network_manager/refresher.rb

```diff
@@ -3,7 +3,11 @@ class Amazon::NetworkManager::Refresher < ManageIQ::Providers::BaseManager::Refr
     include ::EmsRefresh::Refreshers::EmsRefresherMixin
 
     def parse_legacy_inventory(ems)
```

Comment:
Hrm, I'm seeing that this should probably be changed to `collect_and_parse_inventory` ... or, something like that ...

`parse_legacy_inventory` is too nondescript now that we're exploring the enhancements to inventory collection